articles/liquid/liquid.md

```diff
@@ -0,0 +1,126 @@
+# **Exploring Liquid Network’s Bitcoin Sidechain**
+
+## **Introduction**
+
+This article is part of a series exploring the different scaling solutions built on or around Bitcoin's blockchain. The aim is to provide a comprehensive insight into these protocols, highlighting their functionalities, security features, and their role in the broader cryptocurrency ecosystem. In this edition, we will explore the Liquid Network, examining its design, operation, and its implications for Bitcoin's scalability and privacy enhancements.
+
+## **Overview**
+
+Liquid Network is a sidechain-based settlement network that functions as a Bitcoin Layer-2 solution. It interconnects global cryptocurrency exchanges and institutions, facilitating rapid and confidential Bitcoin transactions along with the issuance of digital assets.
+
+Liquid aims to address some of the limitations of the Bitcoin network by providing faster transaction settlements and enhanced privacy, all while maintaining robust security. It enables the issuance of various types of digital assets which opens new avenues for trading and asset management on its platform.
+
+The network was officially launched in 2018 by Blockstream, and it operates as an implementation of Elements, a sidechain-capable blockchain platform. 
+
+![l1](./img/l1.png)
+
+## **Protocol Quick Scan**
```

Comment:
Need to reorganize this part. The content is generally good, but the structure is somewhat not clear